xml/osc_building.xml

```diff
@@ -140,6 +141,62 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   </procedure>
  </sect1>
 
+ <sect1 xml:id="sec.obs.building.advanced">
+  <title>Advanced Build Environment Handling</title>
+  <para>
+	  The default build environment for local builds is usually chroot. While this is simplest environment
+	  and is therefore easy and fast to handle, it has also a number of shortcummings. For one it is
+	  not safe against attacks, so you must not build sources or using build dependendencies from a resource
+	  which you do not trust. Furthermore the environment is not fully isolated and runs not the kernel the
+	  target distribution runs. This means esp image builds and kernel/hardware specific builds may fail or
+	  won't produce the same result. The server side is usually set to inside of KVM therefore to avoid this.
+  </para>
+  <itemizedlist>
+   <listitem>
+     <para>You can build also local in KVM (if your hardware supports it) by running
+      </para>
+      <screen>&prompt.user;<command>osc</command> build --vm-type=kvm --vm-memory=<replaceable>MB</replaceable></screen>
+   </listitem>
+   <listitem>
+      <para>Another important virtualization mode is qemu. This can be used to build for a foreign hardware
+	      architecture even when the distribution is not prepared for the qemu user land emulator. However, this
+	      qemu system emulator approach will be much slower. 
+      </para>
+      <screen>&prompt.user;<command>osc</command> build --vm-type=qemu --vm-memory=<replaceable>MB</replaceable> <replaceable>REPOSITORY</replaceable> <replaceable>ARCHITECTURE</replaceable></screen>
+   </listitem>
+   <listitem>
+     <para>You may want to jump inside of a the build environment for debugging purposes. This can be done via the following
+           </para>
+      <screen>&prompt.user;<command>osc</command> shell --vm-type=<replaceable>VM</replaceable></screen>
+   </listitem>
+   <listitem>
+     <para>A simple way to remove the build environemnt is
+           </para>
+      <screen>&prompt.user;<command>osc</command> wipe --vm-type=<replaceable>VM</replaceable></screen>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1 xml:id="sec.obs.building.spec2rpm">
+  <title>Spec Files (RPM) Specifics</title>
```

Comment:
Specifics -> Details? Maybe (re)move the parenthesis:

```suggestion
  <title>RPM Spec Files Details</title>
```

Reply:
I personally think "specifics" is more accurate since for example macro usage and alike will only work for RPM, but not for kiwi builds.

Reply:
Maybe it's just me, but when I read "specifics" I think more of "specifications". :wink:
When I'm not sure, I usually turn to [Merriam-Webster](https://www.merriam-webster.com/thesaurus/specifics). The Thesaurus defines *specifics* as:

1. a substance or preparation used to treat disease
    // quinine is a *specific* for malaria
1. a single piece of information
  // although the speech was long on rhetoric and platitudinous generalities, it lacked *specifics*
1. something that sets apart an individual from others of the same kind
   // the two submissions differ only in certain *specifics*

Maybe (2) or (3) could fit, but it still looks a bit unusual to me. :smile:  Under (3) the synonyms "characteristics" and "features" are listed. Would that work?

Don't get me wrong, if you think it's still appropriate then use it. I just wanted to help you with the definitions and possible alternatives. :-)

Reply:
> "specifics" I think more of "specifications"

That is you. Not what it means at all.

---

Suggestion: "Functionality Specific to RPM Spec Files"